CHANGELOG.md

```diff
@@ -1,6 +1,354 @@
+    - Fix PICSRC in bash_picongpu.profile #3999
+- Fix that PMacc depends on PIConGPU type definitions #3925
+- HIP: allow full architecture definition #3941
+- Make the span-based storeChunk API opt-in in various openPMD-based IO routines #3933
+- Fix openPMD outputs negative particle patch offset #4037
+- Fix CUDA container apt keys #4085
+- Add HIP and mallocMC + HIP to PIConGPU version output #4139
+- Do not forceinline recursive functions #4147
+- PNG plugin: fix invalid memory access in 2D #4225
+- Fix using uninitilized variable #4303
+- [breaking change] Fix openPMD particles `positionOffset` #4283
+- Fix precision cast is returning a reference #4337
+
+**Misc:**
+- Refactoring:
+    - PIC:
+        - Refactor absorbing and reflecting particle BC #3875
+        - Move yee::StencilFunctor to fdtd::StencilFunctor #3894
+        - Improve performance of MoveParticle #3908
+        - Cleanup current deposition implementation #3944
+        - Use `if constexpr()` to check for simulation/object dimension #4107
+        - Avoid using macro define `SIMDIM` #4116
+        - Switch to alpaka::Complex<> #4114
+        - Add a helper type for unique typelist of enabled incident field profiles #4177
+        - Cached shapes #4124
+        - Use cached shapes for current deposition #4170
+        - Add device side `ForEachParticle` #4173
+        - Workaround wrong reported free memory #4219
+        - Simplify implementation of incident field kernel #4226
+        - Auto adjust particle exchange buffers for double precision #4236
+        - Make adding J term part of field solver #4245
+        - Refactor adding current density to electomagnetic field #4255
+        - Add free temperature functor #4256
+        - Add singleton-like access for AbsorberImpl  #4262
+        - Simplify PmlImpl interface for getting update functors #4266
+        - Remove Cell2Particle #4289
+        - Remove getGlobalSuperCells #4325
+        - Move include collision.unitless to the default loader #4326
+        - Refactor lockstep programming #4321
+        - Switch to std::optional in compute field value #4345
+        - Use cupla/alpaka's min/max function #4350
+        - Remove flylite #4360
+        - Remove using CUSTL for particle creation #4363
+        - Remove Bremsstrahlung and Synchrotron radition extension #4395
+        - Remove CUstl from particle push #4398
+        - Remove cuSTL usage from current deposition #4402
+        - Remove unused vectors transition radiation #4427
+        - Fix incident field calculation order #4451
+        - Improve command line argument validation #4447
+        - Change class RelayPoint to function #4464
+        - OnePositionImpl refactoring #4465
+        - Refactor particle pusher launcher #4471
+        - Refactor differentiation trait #4473
+        - Refactor GetChargeState and GetCharge #4474
+        - Remove usage of `boost::result_of` #4505
+    - PMacc:
+        - Clean/refactor CUSTL #3862
```

Comment:
same as above 
```suggestion
        - Clean/refactor cuSTL #3862
```

Reply:
I removed this line because we deleted cuSTL anyway.